docs/install-kafka-operator.md

```diff
@@ -156,44 +228,97 @@ Install the [Prometheus operator](https://github.com/prometheus-operator/prometh
     --set prometheus.enabled=false
     ```
 
-### Install {{< kafka-operator >}} with Helm {#kafka-operator-helm}
+1. Verify that prometheus-operator has been deployed and is in running state.
+
+    ```bash
+    kubectl get pods -n prometheus
+    ```
+
+    Expected output:
 
-You can deploy {{< kafka-operator >}} using a [Helm chart](https://github.com/banzaicloud/koperator/tree/master/charts). Complete the following steps.
+    ```bash
+    NAME                                                   READY   STATUS    RESTARTS   AGE
+    prometheus-kube-prometheus-operator-646d5fd7d5-s72jn   1/1     Running   0          15m
+    ```
+
+### Install {{< kafka-operator >}} with Helm {#install-kafka-operator-with-helm}
+
+You can deploy {{< kafka-operator >}} using its [Helm chart](https://github.com/banzaicloud/koperator/tree/master/charts). Complete the following steps.
 
 1. Install the {{< kafka-operator >}} CustomResourceDefinition resources (adjust the version number to the {{< kafka-operator >}} release you want to install). This is performed in a separate step to allow you to uninstall and reinstall {{< kafka-operator >}} without deleting your installed custom resources.
 
     ```bash
     kubectl create --validate=false -f https://github.com/banzaicloud/koperator/releases/download/v{{< param "versionnumbers-sdm.koperatorCurrentversion" >}}/kafka-operator.crds.yaml
     ```
 
-1. Add the Banzai Cloud repository to Helm.
+    ```
+    customresourcedefinition.apiextensions.k8s.io/cruisecontroloperations.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkaclusters.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkatopics.kafka.banzaicloud.io created
+    customresourcedefinition.apiextensions.k8s.io/kafkausers.kafka.banzaicloud.io created
+    ```
+
+1. If you are installing {{< kafka-operator >}} on Red Hat OpenShift, you need to elevate the permissions of the namespace you install Koperator and your Kafka resources into.
+
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccounts:kafka
+    ```
+
+    In case you are using a non-default namespace Koperator, you need to modify the command to take that namespace.
 
     ```bash
-    helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com/
-    helm repo update
+    oc adm policy add-scc-to-group anyuid system:serviceaccount:{NAMESPACE_FOR_KOPERATOR}
```

Comment:
`serviceaccounts`

Reply:
Fixed in https://github.com/banzaicloud/koperator-docs/compare/637c34fef67ef34bb8716243a26e30e947cf4a5a..1c81a46b46a8558d88f7a9ddf440d47796031cd0